capabilities.go

```diff
@@ -34,3 +35,14 @@ func (caps *capabilities) isBlockDeviceSupported() bool {
 func (caps *capabilities) setBlockDeviceSupport() {
 	caps.flags = caps.flags | blockDeviceSupport
 }
+
+func (caps *capabilities) isBlockDeviceHotplugSupported() bool {
+	if caps.flags&blockDeviceHotplugSupport != 0 {
+		return true
+	}
+	return false
+}
+
+func (caps *capabilities) setBlockDeviceHotplugSupport() {
+	caps.flags = caps.flags | blockDeviceHotplugSupport
```

Comment:
I guess it's in keeping with the existing code, but I personally find `|=` more intuitive (plus it's less error prone as you only specify the variable once).